### PR TITLE
Jlopezbarb/wait for resources

### PR DESCRIPTION
**Pull request by jLopezbarb**
Waits for resources to be running before getting endpoints

### CODE REVIEW

File: message.go [hunk 1]
```diff
@@ -184,3 +187,52 @@ func getServicesNotRunning(servicesRunningStatus map[string]bool) []string {
 	}
 	return servicesNotRunnig
 }
+
+func waitForResourcesRunning(previewName string) error {
+	retries := 0
+	wait := false
+	for retries < 5 {
```
Comment (pchico83) [inline message.go:194]: Loop until `wait`

File: message.go [hunk 2]
```diff
@@ -184,3 +187,52 @@ func getServicesNotRunning(servicesRunningStatus map[string]bool) []string {
...
+		resourceStatus, err := getResourceStatus(previewName)
+		if err != nil {
+			return err
+		}
+		for _, status := range resourceStatus {
+			if status != "running" {
```
Comment (pchico83) [inline message.go:200]: if `status` is `error` you can fast fail

File: message.go [hunk 3]
```diff
@@ -184,3 +187,52 @@ func getServicesNotRunning(servicesRunningStatus map[string]bool) []string {
...
+				wait = true
+			}
+		}
+		if !wait {
+			break
+		}
+		time.Sleep(10 * time.Second)
```
Comment (pchico83) [inline message.go:207]:
```suggestion
		time.Sleep(5 * time.Second)
```

File: message.go (hunk 2 above)
Comment (jLopezbarb) [reply message.go:200]: what happens if it fails because it does not find the database url and after a retry of the deployment it starts to work?

**Pull request closed without merging**